### PR TITLE
use screen in the blog suggested in live stream

### PR DESCRIPTION
**Pull request by Aprillion**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,6 +29,7 @@ node_modules
 .sass-cache/
 .cache/
 .DS_Store
+.idea
```
Comment (MichaelDeBoey) [inline .gitignore:32]: You should ignore this in your [global `.gitignore`](https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Comment by Aprillion** [comment]
the build failed with a `You must set a GOOGLE_APPLICATION_CREDENTIALS environment variable` message.. which sounds like a problem I will not be able to solve, or I am missed some instructions that I will need help with :)

File: .gitignore (hunk 1 above)
Comment (Aprillion) [reply .gitignore:32]: sure.. do you want me to revert that change on top of ignoring it in my global .gitignore?

Comment (kentcdodds) [reply .gitignore:32]: It's fine, we can leave it here.

**Review by kentcdodds — APPROVED** [review]
Super! Thanks!

**Comment by kentcdodds** [comment]
@all-contributors please add @Aprillion for content

**Pull request merged**